setup.py

```diff
@@ -27,7 +27,6 @@
     "Intended Audience :: Education",
     "Topic :: Scientific/Engineering",
     "Topic :: Software Development :: Libraries",
-    "Programming Language :: Python :: 3.7",
```

Comment:
Could you also change the L36 below to read: `PYTHON_REQUIRES = ">=3.8"`? Thanks!